### ISSUE TITLE
"Open Save Directory" button crashes game on binary packaged for Windows

### ISSUE THREAD

**Issue by NoSkillPureAndy** [FinalForEach/Cosmic-Reach-Issue-Tracker#37]
* Game started: true
* Game version: 0.0.5
* Ran for : 6 seconds, 279 ms
* Current time: 2024-03-08 at 19:22:58.882435600-05:00[America/Indianapolis]
* Operating system: Windows 10 10.0
* Arch: amd64
* Java VM name: OpenJDK 64-Bit Server VM
* Java runtime version: 21.0.2+13-58
* System user language: en
* CPU model: AMD Ryzen 5 3600 6-Core Processor
* Save location: C:\Users\me\AppData\Local\cosmic-reach
* Save location free space: 26.72 GB
* Save location total space: 476.337 GB
* Available processors: 12
* Native heap use: 20 MB
* Java heap use: 20 MB
* Max memory available: 7.984 GB
* RAM available: 31.925 GB
* getGLVersion: 
Type: OpenGL
Version: 4:6:0
Vendor: NVIDIA Corporation
Renderer: NVIDIA GeForce GTX 1660/PCIe/SSE2
* Exception logs: 
com.badlogic.gdx.utils.GdxRuntimeException: java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:173)
    at finalforeach.cosmicreach.lwjgl3.Lwjgl3Launcher.createApplication(Lwjgl3Launcher.java:287)
    at finalforeach.cosmicreach.lwjgl3.Lwjgl3Launcher.main(Lwjgl3Launcher.java:67)
Caused by: java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
    at java.desktop/java.awt.Toolkit.newAWTError(Unknown Source)
    at java.desktop/java.awt.Toolkit.fallbackToLoadClassForAT(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
    at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
    at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
    at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source)
    at java.desktop/java.awt.Toolkit.loadAssistiveTechnologies(Unknown Source)
    at java.desktop/java.awt.Toolkit.getDefaultToolkit(Unknown Source)
    at java.desktop/java.awt.Desktop.isDesktopSupported(Unknown Source)
    at java.desktop/java.awt.Desktop.getDesktop(Unknown Source)
    at finalforeach.cosmicreach.gamestates.MainMenu$2.onClick(MainMenu.java:53)
    at finalforeach.cosmicreach.ui.UIElement.drawBackground(UIElement.java:132)
    at finalforeach.cosmicreach.gamestates.GameState.drawUIElements(GameState.java:81)
    at finalforeach.cosmicreach.gamestates.MainMenu.render(MainMenu.java:157)
    at finalforeach.cosmicreach.BlockGame.render(BlockGame.java:88)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:387)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:193)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:167)
    ... 2 more
Caused by: java.lang.ClassNotFoundException: com.sun.java.accessibility.AccessBridge
    at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
    at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
    at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
    at java.base/java.lang.Class.forName0(Native Method)
    at java.base/java.lang.Class.forName(Unknown Source)
    at java.base/java.lang.Class.forName(Unknown Source)
    ... 24 more

**Comment by Lelebees** [comment]
Ran into the exact same issue! I do want to mention that I've also got several JDK's installed on my system. Attached you find my crash log.

`
* Game started: true
* Game version: 0.0.14
* Ran for : 10 minutes, 10 seconds
* Current time: 2024-03-09 at 18:43:05.055966400+01:00[Europe/Amsterdam]
* Operating system: Windows 11 10.0
* Arch: amd64
* Java VM name: OpenJDK 64-Bit Server VM
* Java runtime version: 21.0.2+13-58
* System user language: en
* CPU model: AMD Ryzen 7 5800H with Radeon Graphics
* Save location free space: 167.29 GB
* Save location total space: 570.607 GB
* Available processors: 16
* Native heap use: 319 MB
* Java heap use: 319 MB
* Max memory available: 3.838 GB
* RAM available: 15.346 GB
* getGLVersion: 
Type: OpenGL
Version: 4:6:0
Vendor: ATI Technologies Inc.
Renderer: AMD Radeon(TM) Graphics
* Exception logs: 
com.badlogic.gdx.utils.GdxRuntimeException: java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:173)
    at finalforeach.cosmicreach.lwjgl3.Lwjgl3Launcher.createApplication(Lwjgl3Launcher.java:290)
    at finalforeach.cosmicreach.lwjgl3.Lwjgl3Launcher.main(Lwjgl3Launcher.java:69)
Caused by: java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
    at java.desktop/java.awt.Toolkit.newAWTError(Unknown Source)
    at java.desktop/java.awt.Toolkit.fallbackToLoadClassForAT(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
    at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
    at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
    at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source)
    at java.desktop/java.awt.Toolkit.loadAssistiveTechnologies(Unknown Source)
    at java.desktop/java.awt.Toolkit.getDefaultToolkit(Unknown Source)
    at java.desktop/java.awt.Desktop.isDesktopSupported(Unknown Source)
    at java.desktop/java.awt.Desktop.getDesktop(Unknown Source)
    at finalforeach.cosmicreach.gamestates.MainMenu$2.onClick(MainMenu.java:54)
    at finalforeach.cosmicreach.ui.UIElement.drawBackground(UIElement.java:133)
    at finalforeach.cosmicreach.gamestates.GameState.drawUIElements(GameState.java:83)
    at finalforeach.cosmicreach.gamestates.MainMenu.render(MainMenu.java:158)
    at finalforeach.cosmicreach.BlockGame.render(BlockGame.java:89)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:387)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:193)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:167)
    ... 2 more
Caused by: java.lang.ClassNotFoundException: com.sun.java.accessibility.AccessBridge
    at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
    at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
    at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
    at java.base/java.lang.Class.forName0(Native Method)
    at java.base/java.lang.Class.forName(Unknown Source)
    at java.base/java.lang.Class.forName(Unknown Source)
    ... 24 more

`

**Comment by FinalForEach** [comment]
Fixed as per a recent update, feel free to reopen / comment if this persists

**Comment by NoSkillPureAndy** [comment]
> Fixed as per a recent update, feel free to reopen / comment if this persists

Nope, same issue as of 0.1.13. Nearly identical stack trace:
```
* Game started: true
* Game version: 0.1.13
* Ran for : 10 seconds, 225 ms
* Current time: 2024-03-25 at 15:15:06.591631600-04:00[America/Indianapolis]
* Operating system: Windows 10 10.0
* Arch: amd64
* Java VM name: OpenJDK 64-Bit Server VM
* Java runtime version: 21.0.2+13-58
* System user language: en
* CPU model: AMD Ryzen 5 3600 6-Core Processor
* Save location free space: 36.275 GB
* Save location total space: 476.337 GB
* Available processors: 12
* Native heap use: 23 MB
* Java heap use: 23 MB
* Max memory available: 7.984 GB
* RAM available: 31.925 GB
* getGLVersion: 
Type: OpenGL
Version: 4:6:0
Vendor: NVIDIA Corporation
Renderer: NVIDIA GeForce GTX 1660/PCIe/SSE2
* Exception logs: 
com.badlogic.gdx.utils.GdxRuntimeException: java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:173)
    at finalforeach.cosmicreach.lwjgl3.Lwjgl3Launcher.createApplication(Lwjgl3Launcher.java:54)
    at finalforeach.cosmicreach.lwjgl3.Lwjgl3Launcher.main(Lwjgl3Launcher.java:44)
Caused by: java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
    at java.desktop/java.awt.Toolkit.newAWTError(Unknown Source)
    at java.desktop/java.awt.Toolkit.fallbackToLoadClassForAT(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
    at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
    at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
    at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
    at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
    at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source)
    at java.desktop/java.awt.Toolkit.loadAssistiveTechnologies(Unknown Source)
    at java.desktop/java.awt.Toolkit.getDefaultToolkit(Unknown Source)
    at java.desktop/java.awt.Desktop.isDesktopSupported(Unknown Source)
    at finalforeach.cosmicreach.io.SaveLocation.OpenFolderWithFileManager(SaveLocation.java:20)
    at finalforeach.cosmicreach.gamestates.MainMenu$2.onClick(MainMenu.java:53)
    at finalforeach.cosmicreach.ui.UIElement.drawBackground(UIElement.java:143)
    at finalforeach.cosmicreach.gamestates.GameState.drawUIElements(GameState.java:87)
    at finalforeach.cosmicreach.gamestates.MainMenu.render(MainMenu.java:163)
    at finalforeach.cosmicreach.BlockGame.render(BlockGame.java:114)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:387)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:193)
    at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:167)
    ... 2 more
Caused by: java.lang.ClassNotFoundException: com.sun.java.accessibility.AccessBridge
    at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
    at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
    at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
    at java.base/java.lang.Class.forName0(Native Method)
    at java.base/java.lang.Class.forName(Unknown Source)
    at java.base/java.lang.Class.forName(Unknown Source)
    ... 24 more
```

**Comment by FinalForEach** [comment]
After examining the logs, I've adjusted the catch to fix it, it should be fixed for 0.1.14
